pkg/operator/controller/names.go

```diff
@@ -1,5 +1,34 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package nodeobservabilitycontroller
 
+import "k8s.io/apimachinery/pkg/types"
+
 const (
-	AgentName = "node-observability-agent"
+	// nodeObservabilityBaseName  is the base name for any NodeObservability resource.
+	kubeletCM      = "kubelet-serving-ca"
+	ControllerName = "node_observability_controller"
+	AgentName      = "node-observability-agent"
 )
+
+// NodeObsKubeletCAConfigMapName returns the namespaced name of the destination (operand) kubelet CA configmap
+func NodeObsKubeletCAConfigMapName(operandNamespace string) types.NamespacedName {
+	return types.NamespacedName{
+		Namespace: operandNamespace,
+		Name:      kubeletCM,
+	}
+}
```

Comment:
I don't quite see the helpfulness of this function, I think it makes more sense to expose kubelet configMap name and use it with whatever namespace is needed.

Reply:
that's what this function is doing, and it's also used by the ca-configmap controller.